File: Durwapatil/C++ Programs/addition.cpp
// Date : 24 Jan 2021

// CPP program to display the addition of two numbers in CPP



            #include <iostream>
            using namespace std;

            class ab
            {
              private : int a,b,c;
              public : int add();
            };
            int ab::add()
            {
              cout << "Enter two numbers : \n";
              cin >> a >> b;
              c = a + b;
              cout << "Sum : " << c;
            }
            int main()
            {
              ab st;
              st.add();
            }



/*
Output : 
    Enter two numbers : 
    23
    52
    
    Sum : 75
*/

File: Durwapatil/Java/addition.java
// Date : 25 Jan 2021 

// Java program to display the addition of two numbers


            import java.io.*;
            class add
            {
              public static void main(String args[])
              {
                try
                {
                  DataInputStream x = new DataInputStream(System.in);
                  int a, b, c;
                  System.out.print("Enter 2 numbers : \n");
                  a = Integer.parseInt(x.readLine());
                  b = Integer.parseInt(x.readLine());
                  c = a + b;
                  System.out.print("\nThe addition is  "+ c);
                }
                catch(Exception e)
                {}
              }
            }
            
/*
Output : 
    Enter two numbers : 
    5
    3
    
    The addition is 8
*/
